README.md

```diff
@@ -251,3 +251,32 @@ As mentioned in the linked guide you can run terraform init Initialize the proje
 
 However before running the apply command you can also run terraform plan to verify that the installation. 
 
+## Module 13 ~ Generating extra Logs and send to Loggly
+
+For this section we will be usuing the logger module to log the app activities and send those logs to loggly.
+
+For a tutorial on the logging module, please see the link below:
+
+https://docs.python.org/3/howto/logging.html
+
+
+For documentation on sending the logs to Loggly using HTTPS, please see the link below:
+
+https://documentation.solarwinds.com/en/success_center/loggly/content/admin/python-http.htm?cshid=loggly_python-http
+
+
+To be able to access a logger object from inside the app you can use a syntax of:
+
+app.logger.info("Value of foo is %s", foo)
+
+For more precise examples please see lines 31, 47, 53, 59, 76 and 113 in the file app.py and line 31 and 39 from the file mongo_items_collect.py, also please note that the logger statements have been placed in the code blocks where what they are logging are defined. 
```

Comment:
This sort of comment, while seemingly helpful, is _very_ likely to be the opposite; the next time you place an import statement into app.py, you have made this comment invalid, and will have to update it. In one of my strongest review statements -I _guarantee_ that you'll either forget or you'll delete this comment to save updating it within a handful of changes! So I'd pre-empt that, and remove this - the info above gives them an idea, and they can then search the project for `app.logger` to find specific examples